packages/react-components/react-tree/library/src/components/FlatTree/FlatTree.cy.tsx

```diff
@@ -434,7 +434,7 @@ describe('FlatTree', () => {
         />,
       );
       cy.window().then(win => {
-        expect(win.console.error).to.be.callCount(1);
+        expect(win.console.error).to.be.called;
```

Comment:
As StrictMode renders twice, this callCount was also two. Just making sure the console.error is called is sufficient here.